src/django/api/models/submission.py

```diff
@@ -0,0 +1,87 @@
+from django.db import models
+from django.core.exceptions import ValidationError
+from django.contrib.gis.db import models as gis_models
+
+from .boundary import Boundary
+from .role import Roles
+from .user import User
+
+__all__ = ["Submission", "Review", "Approval", "Annotation"]
+
+
+def limit_by_validator_or_admin():
+    return models.Q(role__pk=Roles.VALIDATOR) | models.Q(role__pk=Roles.ADMINISTRATOR)
+
+
+class Submission(models.Model):
+    boundary = models.ForeignKey(Boundary, on_delete=models.CASCADE)
+    shape = gis_models.PolygonField(geography=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    created_by = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name="creator"
+    )
+    submitted_at = models.DateTimeField(null=True, blank=True)
+    submitted_by = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        blank=True,
+        null=True,
+        related_name="submitter"
+    )
+    upload_filename = models.CharField(max_length=100, blank=True)
+    upload_edited_at = models.DateTimeField(null=True, blank=True)
+    notes = models.TextField(blank=True)
```

Comment:
Add `null=True`

Reply:
I left off `null=True` for the `TextField` and `CharField` so that the value would be an empty string vs NULL, is there a benefit to using NULL here?

Reply:
NULL is a clearer representation of an empty value. Without it, we'll have to manually specify an empty string when creating a new submission. I'd prefer to have it.

Reply:
See updated discussion at: https://azavea.slack.com/archives/C02N5S81A69/p1664466431686839?thread_ts=1664466073.244999&cid=C02N5S81A69